pkg/reconciler/taskrun/taskrun.go

```diff
@@ -144,7 +144,7 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, tr *v1beta1.TaskRun) pkg
 
 	// If the TaskRun is cancelled, kill resources and update status
 	if tr.IsCancelled() {
-		message := fmt.Sprintf("TaskRun %q was cancelled", tr.Name)
+		message := fmt.Sprintf("TaskRun %q was cancelled. %s", tr.Name, tr.GetStatusMessage())
```

Comment:
I think we'll want this message to be propagated to the `taskRun.status.conditions[].message`

Reply:
@lbernick This gets propagated to taskRun.status.conditions[].message. This like does the propagation. Also test file pkg/reconciler/taskrun/taskrun_test.go has a test which checks for this propagation.

Reply:
you're right! sorry I missed that.